app/config.js

```diff
@@ -201,7 +201,7 @@ tag.listView()
         }),
         nga.field('custom', 'template')
             .label('Upper name')
-            .template('{{ entry.values.name.toUpperCase() }}')
```

Comment:
I didn't find any solution for the moment. I passed to pagination to go forward.